### PR TITLE
Add Release notes for 1.10

### LINKED ISSUE #1304: Release Notes for 1.10
- Implemented new address bar layout. ([#1134](https://github.com/brave/brave-ios/issues/1134))
 - Implemented new main menu layout. ([#1130](https://github.com/brave/brave-ios/issues/1130))
 - Added ability to "Find in page" using the URL bar. ([#1019](https://github.com/brave/brave-ios/issues/1019))
 - Added 1Password activity in share sheet. ([#948](https://github.com/brave/brave-ios/issues/948))
 - Added image titles when using long-press on images. ([#851](https://github.com/brave/brave-ios/issues/851))
 - Added privacy warning on Brave Sync code page. ([#1225](https://github.com/brave/brave-ios/issues/1225))
 - Improved swipe gesture on bookmark hanger. ([#953](https://github.com/brave/brave-ios/issues/953)) 
 - Improved scrolling issue on websites with sticky headers and footers. ([#631](https://github.com/brave/brave-ios/issues/631))
 - Improved icon resolution for favourites. ([#463](https://github.com/brave/brave-ios/issues/463))
 - Fixed several random crashes in certain conditions. ([#1120](https://github.com/brave/brave-ios/issues/1120))
 - Fixed "Block-all-cookies" toggle behaviour. ([#897](https://github.com/brave/brave-ios/issues/897))
 - Fixed not being able to enter passcode after lockout timer. ([#938](https://github.com/brave/brave-ios/issues/938))
 - Fixed browser failing to start on iPad Air2 in certain conditions. ([#1040](https://github.com/brave/brave-ios/issues/1040)) 
 - Fixed browser failing to launch when device uses Swedish as default language. ([#1111](https://github.com/brave/brave-ios/issues/1111))
 - Fixed auto-focus of a tab when a link is opened using the share sheet. ([#698](https://github.com/brave/brave-ios/issues/698))
 - Fixed missing translations when viewing introduction summary under private tab. ([#1239](https://github.com/brave/brave-ios/issues/1239))
 - Fixed webcompat issues with https://borsen.dk/ due to Brave shields. ([#1061](https://github.com/brave/brave-ios/issues/1061))

### PR DESCRIPTION
**Pull request by srirambv**
Fixes #1304

### CODE REVIEW

**Review by kjozwiak — APPROVED** [review]
LGTM 👍 @jhreis mind taking a luck as well in case we've missed something?

**Review by jhreis — CHANGES_REQUESTED** [review]
We don't know what will be included in 1.10, so blocking until this is finalized.

**Comment by srirambv** [comment]
@jhreis updated release notes. Ready for re-review

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ - Added ability to "Find in page" using the URL bar. ([#1019](https://github.com/brave/brave-ios/issues/1019))
+ - Added 1Password activity in share sheet. ([#948](https://github.com/brave/brave-ios/issues/948))
+ - Added image titles when using long-press on images. ([#851](https://github.com/brave/brave-ios/issues/851))
+ - Added privacy warning on Brave Sync code page. ([#1225](https://github.com/brave/brave-ios/issues/1225))
+ - Improved swipe gesture on bookmark hanger. ([#953](https://github.com/brave/brave-ios/issues/953)) 
+ - Improved scrolling issue on websites with sticky headers and footers. ([#631](https://github.com/brave/brave-ios/issues/631))
+ - Improved icon resolution for favourites. ([#463](https://github.com/brave/brave-ios/issues/463))
+ - Fixed several random crashes in certain conditions. ([#1120](https://github.com/brave/brave-ios/issues/1120))
+ - Fixed "Block-all-cookies" toggle behaviour. ([#897](https://github.com/brave/brave-ios/issues/897))
+ - Fixed not being able to enter passcode after lockout timer. ([#938](https://github.com/brave/brave-ios/issues/938))
```
Comment (kjozwiak) [inline CHANGELOG.md:16]: @srirambv did this one make it in as #1203 didn't? I'm not sure if those two fixes were fixed at the same time.

Comment (srirambv) [reply CHANGELOG.md:16]: Both are different issues and different fixes. #1203 fix is part of 1.11 now

**Review by jhreis — APPROVED** [review]

**Pull request merged**